plugins/services/src/js/components/forms/PodContainerServiceFormSection.js

```diff
@@ -0,0 +1,249 @@
+import React, {Component} from 'react';
+import {Tooltip} from 'reactjs-components';
+
+import {FormReducer as ContainerReducer} from '../../reducers/serviceForm/Container';
+import {FormReducer as ContainersReducer} from '../../reducers/serviceForm/Containers';
+import AdvancedSection from '../../../../../../src/js/components/form/AdvancedSection';
+import AdvancedSectionContent from '../../../../../../src/js/components/form/AdvancedSectionContent';
+import AdvancedSectionLabel from '../../../../../../src/js/components/form/AdvancedSectionLabel';
+import DeleteRowButton from '../../../../../../src/js/components/form/DeleteRowButton';
+import FieldError from '../../../../../../src/js/components/form/FieldError';
+import FieldHelp from '../../../../../../src/js/components/form/FieldHelp';
+import FieldInput from '../../../../../../src/js/components/form/FieldInput';
+import FieldLabel from '../../../../../../src/js/components/form/FieldLabel';
+import FieldTextarea from '../../../../../../src/js/components/form/FieldTextarea';
+import FormGroup from '../../../../../../src/js/components/form/FormGroup';
+import MetadataStore from '../../../../../../src/js/stores/MetadataStore';
+import Icon from '../../../../../../src/js/components/Icon';
+import {findNestedPropertyInObject} from '../../../../../../src/js/utils/Util';
+
+class PodContainerServiceFormSection extends Component {
+  getAdvancedSettings(data = {}, errors = {}) {
```

Comment:
Please align this section with the single container implementation, which is alway rendering the "Artifact URI" label.